app/models/manageiq/providers/amazon/storage_manager/ebs/cloud_volume.rb

```diff
@@ -96,6 +104,74 @@ def provider_object(connection = nil)
     connection.volume(ems_ref)
   end
 
+  def self.params_for_create(ems_id)
+    {
+      :fields => [
+        {
+          :component  => 'select',
+          :name       => 'availability_zone_id',
+          :id         => 'availability_zone_id',
+          :label      => _('Availability Zone'),
+          :options    => ManageIQ::Providers::Amazon::CloudManager::AvailabilityZone.where(:ems_id => ems_id).map do |az|
+            {
+              :label => az.name,
+              :value => az.id,
+            }
+          end,
+          :isRequired => true,
+          :validate   => [{:type => 'required'}],
+        },
+        {
+          :component    => 'select',
+          :name         => 'volume_type',
+          :id           => 'volume_type',
+          :label        => _('Cloud Volume Type'),
+          :options      => CLOUD_VOLUME_TYPES.map do |value, label|
+            {
+              :label => _(label),
+              :value => value,
+            }
+          end,
+          :isRequired   => true,
+          :validate     => [{:type => 'required'}],
+          :initialValue => 'gp2',
+        },
+        {
+          :component => 'text-field',
+          :name      => 'iops',
+          :id        => 'iops',
+          :label     => _('IOPS'),
+          :type      => 'number',
+          :condition => {
+            :when => 'volume_type',
+            :is   => 'io1',
+          }
+        },
+        {
+          :component => 'select',
+          :name      => 'cloud_volume_snapshot_id',
+          :id        => 'cloud_volume_snapshot_id',
+          :label     => _('Base Snapshot'),
+          :options   => ManageIQ::Providers::Amazon::StorageManager::Ebs::CloudVolumeSnapshot.where(:ems_id => ems_id).map do |cvs|
```

Comment:
@skateman we can't use the same ems_id to query for availability zones in the [CloudManager :point_up:](https://github.com/ManageIQ/manageiq-providers-amazon/pull/657/files#diff-89fd56220c0624c072f49be1ec3b9b06153377a98664a770ed5723413b18216eR115) and in the `StorageManager`.

We do delegate availability zones to the parent cloud manager so I think what we should do here is:
```
storage_manager = ManageIQ::Providers::Amazon::StorageManager::Ebs.find_by(:id => id)
storage_manager.availability_zones # this will delegate to the parent cloud_manager automatically
storage_manager.cloud_volume_snapshots
```